test/integration_test.go

```diff
@@ -437,7 +443,7 @@ func (ts *IntegrationTestSuite) TestChildWFWithParentClosePolicyAbandon() {
 	ts.NoError(err)
 	resp, err := ts.libClient.DescribeWorkflowExecution(context.Background(), childWorkflowID, "")
 	ts.NoError(err)
-	ts.True(resp.WorkflowExecutionInfo.GetCloseTime() == 0)
+	ts.Truef(resp.WorkflowExecutionInfo.GetCloseTime() == 0, "Expected close time to be zero, got %d. Describe response: %+v", resp.WorkflowExecutionInfo.GetCloseTime(), resp)
```

Comment:
```suggestion
	ts.Zerof(resp.WorkflowExecutionInfo.GetCloseTime(), "Child workflow should not be closed, describe response: %+v", resp)
```
possibly %#v too, but if + looks better here then definitely 👍 .  I just find + to be pretty rarely supported, and the %v it degrades to is pretty bad for some structures.